src/yashCJun20/CharOccurance.java

```diff
@@ -0,0 +1,32 @@
+package yashCJun20;
+
+public class CharOccurance {
+	
+	void Charindex(String name) {
+		
+		for(int i=0;i<name.length();i++)
+		{
+		char ch=name.charAt(i);
+		if(i == name.indexOf(ch) && i!=name.lastIndexOf(ch)){
+			System.out.print(ch+" -> first index = "+i);
+		for(int index=0;index<name.length();index++) {
+			if(index==name.lastIndexOf(ch)){
+				System.out.println(" last index = "+index);
```

Comment:
You have increased the complexity in code, Using one for loop and one if block you can achieve the output. 